pkg/sfu/downtrack.go

```diff
@@ -345,9 +341,10 @@ func (d *DownTrack) writeSimpleRTP(extPkt *buffer.ExtPacket) error {
 			}
 		}
 
-		d.snOffset = extPkt.Packet.SequenceNumber - d.lastSN - 1
-		d.tsOffset = extPkt.Packet.Timestamp - d.lastTS - 1
-
+		if d.lastSN != 0 {
```

Comment:
with sync we always set timestamps to 1 and increment from there according to the offsets, this will just set the first timestamp the same as the source and just make sure that we follow spec which says that TS and SN should be random.